.github/workflows/ci-ubuntu-combined.yml

```diff
@@ -8,23 +8,23 @@ on:
       - '*'
   pull_request:
     branches:
-      - master
-      - main
+      - '*'
 
 env:
-  IDRIS2_VERSION: 0.3.0 # For previous-version build
+  IDRIS2_VERSION: 0.4.0 # For previous-version build
```

Comment:
Sorry, but why is this bumped to 0.4.0? This should be the _previous version_, no? Otherwise, the comment is misleading and should be removed.

Reply:
Though a better name might be `latest-version`

Reply:
Ah, my bad. Thanks for the clarification.

Reply:
No worries, though not sure if I should rename the jobs too to make that comment clearer.